about.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
```

Comment:
Remove the subtitle from the about page since there isn't any on the template. 
Fix the color of the text since it is really hard to read it in this font color. 
Provide the border on the box to match the one from the template. 

![Screenshot from 2023-08-02 15-15-29](https://github.com/memicelis/web-development-bootcamp/assets/118063058/f2997485-16a5-45a9-aa36-17b70e6f80d4)

![Screenshot from 2023-08-02 15-15-37](https://github.com/memicelis/web-development-bootcamp/assets/118063058/c3ebf438-9d9c-4fc2-b171-d0d4f42ee4ea)